docs/specifications/tests/DNSQueryAndResponseDefaults.md

```diff
@@ -0,0 +1,162 @@
+the default value cannot be overidden.
+
+|Setting Item |Default value |Fixed |Comment                       |
+|:------------|:-------------|:-----|:-----------------------------|
+|Protocol     | UDP          |      |                              |
+|OpCode       | "query"      | yes  |                              |
+|QR flag      | unset        | yes  | Always unset in a query      |
+|AA flag      | unset        | yes  |                              |
+|TC flag      | unset        | yes  |                              |
+|RD flag      | unset        |      |                              |
+|RA flag      | unset        | yes  |                              |
+|AD flag      | unset        |      |                              |
+|CD flag      | unset        |      |                              |
+|RCODE        | "NoError"    | yes  |                              |
+|Query name   | -            |      | Must be defined in test case |
+|Query type   | -            |      | Must be defined in test case |
+|Query class  | "IN"         | yes  |                              |
+|EDNS         | no           |      | No OPT record is included    |
+
+
+## Default setting in *EDNS Query*
+
+An *EDNS Query* inherit the default setting from a *DNS Query* except for the
+setting items specified below. If a *Setting Item* is specified as "fixed" then
+the default value cannot be overidden.
+
+A test case specification can refer to an *EDNS Query* with one or several
+changes to the setting items.
+
+|Setting Item          |Default value        |Fixed |Comment |
+|:---------------------|:--------------------|:-----|:-------|
+|EDNS                  | OPT record included | yes  |        |
+|EDNS UDP Message size | 512                 |      |        |
+|EDNS Extended RCODE   | no                  |      |        |
+|EDNS Version          | 0                   |      |        |
+|EDNS DO flag          | unset               |      |        |
+|EDNS Z lag            | unset               |      |        |
+|EDNS0 Option          | none                |      |        |
+
+
+## Default setting in *DNSSEC Query*
+
+An *DNSSEC Query* inherit the default setting from an *EDNS Query* except for the
+setting items specified below. If a *Setting Item* is specified as "fixed" then
+the default value cannot be overidden.
+
+A test case specification can refer to an *DNSSEC Query* with one or several
+changes to the setting items.
+
+|Setting               |Default value |Fixed |Comment |
+|:---------------------|:-------------|:-----|:-------|
+|EDNS DO flag          | set          | yes  |        |
+
+
+## Default handling of a *DNS Response*
+
+A *DNS Response* is a response to a *DNS Query*. Unless specified in the test
+case specification, the items in the response is handled as listed. If a
+*Response Item* is specified as "fixed" then the requirement in default action
+must be successful for the response to be considered to be a valid DNS response.
```

Comment:
suggestion: could you  rephrase the last sentence? (using `default action` seems confusing because it does not refer to any part of the following array)

> If a *Response Item* is specified as "fixed" then the default handling must be successful for the response to be considered to be a valid DNS response.

Reply:
Rephrased. Please re-review.